packaging/debian/changelog

```diff
@@ -1,4 +1,4 @@
-google-guest-agent (1:20191009.00-g1) stable; urgency=medium
+google-guest-agent (1:20191021.00-g1) stable; urgency=medium
```

Comment:
will allow this...this time >:)

Reply:
Yeah I know... :p this will mostly be irrelevant when we have auto versioning I hope